Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -306,6 +318,7 @@ def bottle_formula(f)
       bottle.prefix prefix
     end
     bottle.rebuild rebuild
+    bottle.compression_type compression_type
```

Comment:
I have a minor question here. It's reasonably common when new major versions of macOS arrive to mass bottle `homebrew/core` but _not_ replace the old bottles at the same time, rather simply chucking a new one onto the pile. What happens with this there? Because you'll inevitably for quite some time end up with a mixture of `gz` and `xz` bottles. You wouldn't want to have the `compression_type` set to `xz` for all the bottles in that situation, presumably?

Reply:
https://github.com/kabel/brew/blob/142204dc4e7477f6b5500971982b49aa0c40ce7d/Library/Homebrew/dev-cmd/bottle.rb#L327 should be tweaked to check for mismatches here